hdp/src/preprocessor/module_registry.rs

```diff
@@ -62,7 +65,7 @@ impl ModuleRegistry {
             program_hash.map(|program_hash| FieldElement::from_hex_be(&program_hash).unwrap());
         let module_inputs = module_inputs
             .into_iter()
-            .map(|input| FieldElement::from_hex_be(&input).unwrap())
+            .map(|input| ModuleInput::from_str(&input).unwrap())
```

Comment:
lets remove unwrap here
let map do the vec of results then collect it into result of vec and ? it